scripts/bash/enceladus_env.template.sh

```diff
@@ -34,3 +34,6 @@ CONF_CLASS="za.co.absa.enceladus.conformance.DynamicConformanceJob"
 CONF_JAR="enceladus-conformance.jar"
 
 LOG_DIR="/tmp"
+
+# Additional environment-specific Spark options, e.g. "spark.driver.host=myhost"
+ADDITIONAL_SPARK_CONF=""
```

Comment:
This is a dummy question, but this can be set by export right? The empty string won't overwrite that?

Excuse my "bashgnorance" 😄

Reply:
Double checked - the empty string will overwrite the variable even if it is already set by `export`. So this should be safe against users trying to inject arbitrary options outside of `enceladus_env.sh` 😄

Reply:
Thanks for the concern - every concern should be checked.